Makefile

```diff
@@ -43,6 +45,15 @@ deploy-observatorium-loki: download-observatorium-loki-template
 	oc process -f $(LOKI_TEMPLATE_FILE) -p NAMESPACE=$(LOKI_NAMESPACE) -p LOKI_S3_SECRET=test --param-file $(LOKI_CONFIG_FILE) | oc -n $(LOKI_NAMESPACE) apply -f -
 .PHONY:deploy-observatorium-loki
 
+deploy-s3-bucket:
+	hack/create-s3-bucket.sh $(LOKI_STORAGE_BUCKET)
```

Comment:
@Red-GV make sense to add to default LOKI_STORAGE_BUCKET the username so that we will not overlap with the default name `/loki-benchmark-storage`